### ISSUE TITLE
0.21.0 or earlier will not build on bazel 0.27

### ISSUE THREAD

**Issue by BrentDouglas** [bazelbuild/rules_sass#89]
When I try and include rules_sass 1.21.0 (or earlier) on bazel 0.27 I get the following error:

```
INFO: Analyzed target //:site (124 packages loaded, 1571 targets configured).
INFO: Found 1 target...
WARNING: /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/external/io_bazel_rules_sass/sass/BUILD:11:1: runfiles symlink external/io_bazel_rules_sass/sass/sass/sass.js -> external/io_bazel_rules_sass/sass/sass/sass.js obscured by external/io_bazel_rules_sass/sass/sass -> bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass
ERROR: missing input file '@io_bazel_rules_sass//sass:sass/sass.js'
ERROR: /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/external/io_bazel_rules_sass/sass/BUILD:11:1: Creating runfiles tree bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles [for host] failed: build-runfiles failed: error executing command 
  (cd /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/execroot/io_machinecode_hexane && \
  exec env - \
    PATH=/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/usr/local/sbin:/home/brent/.cargo/bin:/usr/local/sbin:/home/brent/.cargo/bin \
  /home/brent/.cache/bazel/_bazel_brent/install/756a8aa28dcd48a791cc43513b17974d/_embedded_binaries/build-runfiles bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles_manifest bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles): Process terminated by signal 15: Process terminated by signal 15
ERROR: /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/external/io_bazel_rules_sass/sass/BUILD:11:1: @io_bazel_rules_sass//sass:sass: missing input file '@io_bazel_rules_sass//sass:sass/sass.js'
Target //:site failed to build
ERROR: /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/external/io_bazel_rules_sass/sass/BUILD:11:1 1 input file(s) do not exist
INFO: Elapsed time: 48.904s, Critical Path: **0.12s**
```

If I look in the dir the warning is complaining you can see that sass/sass.js can't really be a thing there.

```
[2019-06-24 16:51:37 MCLT001 ~/work/hexane (master)]$ ll bazel-out/host/bin/external/io_bazel_rules_sass/sass/
total 368
lrwxrwxrwx 1 brent brent    166 Jun 24 15:11 sass -> /home/brent/.cache/bazel/_bazel_brent/0f791a3a7060acaa1e82ffd53604d76f/execroot/io_machinecode_hexane/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass_bin.sh
-r-xr-xr-x 1 brent brent  19844 Jun 24 16:37 sass_bin_loader.js
-r-xr-xr-x 1 brent brent   5849 Jun 24 15:11 sass_bin.sh
drwxr-xr-x 8 brent brent    178 Jun 24 16:49 sass.runfiles
-r-xr-xr-x 1 brent brent 347110 Jun 24 16:37 sass.runfiles_manifest

```

This was fixed here https://github.com/bazelbuild/rules_sass/commit/9a00e557c32ac0f26e5d3c66d1d17316874027b3#diff-f9e6cfe351e7bb27a86c9de4f3b688da when the entrypoint was qualified, but this is not in a release yet and the error message and its doesn't really line up with the commit message so its not super easy to tell whats going on. Switching the import to 86ca977cf2a8ed481859f83a286e164d07335116 fixed this issue for me, but it would be nice to have a release version that worked.

Is there a release planned any time soon?

**Comment by nex3** [comment]
Typically releases are cut along with new Sass versions. I'm planning to release a new version of Dart Sass once https://github.com/sass/dart-sass/pull/724 lands.
